css-scroll-anchoring-1/Overview.bs

```diff
@@ -85,16 +86,41 @@ An anchor node can be any <a>box</a> except one for a non-<a>atomic inline</a>.
 The anchor node is always a <a>descendant</a> of the <a>scrolling box</a>.
 In some cases, a scrolling box may not select any anchor node.
 
+An element |C| is a <dfn id="anchor-viable-candidate">viable candidate</dfn>
+for becoming a scroll anchor for a scrolling box |S| if it meets all of the
+following criteria:
+  * |C| is a non-<a>atomic inline</a>.
```

Comment:
This doesn't make sense. Atomic inlines are fine, it's non-atomic-inlines the ones that can't be anchors.

Reply:
Oh you're absolutely right, the text I added has the negated meaning of atomic/non-atomic inlines. I fixed it up.